core/src/main/java/org/geysermc/geyser/util/EntityUtils.java

```diff
@@ -162,11 +162,13 @@ public static void updateMountOffset(Entity passenger, Entity mount, boolean rid
                 }
                 case CHICKEN -> zOffset = -0.1f;
                 case TRADER_LLAMA, LLAMA -> zOffset = -0.3f;
+                case ARMOR_STAND -> yOffset += 0.5f;
```

Comment:
This doesn't seem to be correct as it causes all passengers to float above the armor stand. Is there a specific passenger that this fixes?

Master:
![master](https://user-images.githubusercontent.com/4722249/168706096-0e2fab93-3776-4537-b919-a8680c9d92e1.png)
Changes
![changes](https://user-images.githubusercontent.com/4722249/168706114-be613c3f-33ed-48b5-a4b5-09893880fe2a.png)
: